app/mails/management/commands/import.py

```diff
@@ -50,47 +50,24 @@ def import_mail(self, message):
 
             return
 
-        if account.anti_spam:
-            if not len(keys):
-                message.delete()
-                logger.error("Mail from %s deleted: No key" % sender)
-                tools.send_wrong_recipient_mail(sender)
-
-                return
-            elif not any(key == account.key for key in keys):
-                message.delete()
-                logger.error("Mail from %s deleted: Wrong key" % sender)
-
-                return
+        handle_anti_spam_account(
```

Comment:
You need to `return` from this method if `handle_anti_spam_account()` deleted the message, otherwise you'll fall over on the next steps

Reply:
done